app/mailers/quote_mailer.rb

```diff
@@ -1,11 +1,32 @@
 class QuoteMailer < ApplicationMailer
   def new_quote(quote)
     @quote = quote
+    base_URL = "https://maps.googleapis.com/maps/api/staticmap?"
+    polygon_options = "path=color:0x61D5DD|fillcolor:0x61D5DD|weight:5|"
+    polygons_strings = build_coordinates_strings
+    polygons = polygons_strings.map { |string| polygon_options + string}.join("&")
+    zoom = "20"
+    size = "512x512"
+    map_type = "satellite"
+    api_key = "AIzaSyDafFObZ3lBKKVKjYwxGL2xcvqVEwC68XY"
+    @static_map_URL = "#{base_URL}#{polygons}&zoom=#{zoom}&size=#{size}&maptype=#{map_type}&key=#{api_key}"
```

Comment:
I would move this to a service or a helper (for example, you'll probably want to use this in the back-office to show the area selected